partials/tabs/getinvolved.html

```diff
@@ -77,3 +77,9 @@ <h4 class="light center">We’d love to share a bar of chocolate with you!
     </div>
 </div>
 </section>
+<script>
+ $(document).ready(function(){
+    $('ul.tabs').tabs();
+});
+
+</script>
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/getinvolved.html
+++ b/tmp/tmp8v872ato/partials/tabs/getinvolved.html
@@ -82,4 +82,4 @@
     $('ul.tabs').tabs();
 });
 
-</script>+</script>
```